### ISSUE TITLE
opacity rule not reset on compton restart

### ISSUE THREAD

**Issue by fleutot** [chjj/compton#387]
1 . run compton with 
`opacity-rule = [ "50:class_d = 'Gnome-terminal'" ];`
2 . kill compton
3 . restart compton with `opacity-rule = [ "100:class_d = 'Gnome-terminal'" ];`
-> opacity is still 50 (should be "unset"?)

I am looking for a way to return to full opacity on focus, after having tested 50. 

4 . kill compton
5 . restart compton with `opacity-rule = [ "99:class_d = 'Gnome-terminal'" ];`
-> opacity is 99, but inactive terminals are fully opaque, `inactive-opacity` is disregarded.

6 . kill compton
7 . remove the opacity-rule for Gnome-terminal
-> opacity is still 99, but inactive terminals are fully opaque, `inactive-opacity` is disregarded.

In other words:
- 100 does not unset previously set opacity rule
-  inactive-opacity does not work after I have fiddled with opacity-rules, even if I kill and restart compton.

**Comment by fleutot** [comment]
Apparently this issue only appears on already existing windows. New gnome-terminal windows behave as expected.

Of course this is not critical, but it confused me and made me spend quite some time on it.

**Comment by Rostgnom** [comment]
Would anyone happen to know how to apply compton to the already existing windows aswell?
